osbenchmark/aggregator.py

```diff
@@ -169,6 +169,7 @@ def build_aggregated_results(self):
         test_procedure = loaded_workload.find_test_procedure_or_default(test_exe.test_procedure)
 
         test_execution = metrics.create_test_execution(self.config, loaded_workload, test_procedure, test_exe.workload_revision)
+        test_execution.user_tags = list(self.test_executions.keys())
```

Comment:
Alternatively, we could assign the user_tag to a k,v pair something like
```
 test_execution.user_tags = {
    "aggregation-of-runs": list(self.test_executions.keys())
}
```

Reply:
Would help users who are trying to filtering for specific aggregated files in the MDS

Reply:
I didn't consider this, I fixed it to look like your suggestion above!